apps/backoffice-v2/src/pages/Entity/components/Case/hooks/useCaseActionsLogic/useCaseActionsLogic.tsx

```diff
@@ -57,7 +57,38 @@ export const useCaseActionsLogic = ({ workflowId, fullName }: IUseActions) => {
 
   // Avoid passing the onClick event to mutate
   const onMutateApproveEntity = useCallback(() => mutateApproveEntity(), [mutateApproveEntity]);
-  const onMutateRevisionCase = useCallback(() => mutateRevisionCase(), [mutateRevisionCase]);
+  const childPendingWorkflowEvents = useMemo(
+    () =>
+      workflow?.childWorkflows?.flatMap(
+        childWorkflow => childWorkflow?.context?.pendingWorkflowEvents,
+      ) ?? [],
+    [workflow?.childWorkflows],
+  );
+  const pendingWorkflowEvents = useMemo(() => {
+    return [...childPendingWorkflowEvents, ...(workflow?.context?.pendingWorkflowEvents ?? [])];
+  }, [childPendingWorkflowEvents, workflow?.context?.pendingWorkflowEvents]);
+  const onMutateRevisionCase = useCallback(() => {
+    pendingWorkflowEvents?.forEach(pendingWorkflowEvent => {
+      if (pendingWorkflowEvent?.event !== 'revision') {
+        return;
+      }
+
+      mutateRevisionCase({ workflowId: pendingWorkflowEvent?.workflowId });
+
+      const isKybExampleVariant = checkIsKybExampleVariant({
+        variant: workflow?.workflowDefinition?.variant,
+        config: workflow?.workflowDefinition?.config,
+        version: workflow?.workflowDefinition?.version,
+      });
+
+      if (!isKybExampleVariant || pendingWorkflowEvent?.workflowId !== workflow?.id) return;
+
+      window.open(
+        `${workflow?.context?.metadata?.collectionFlowUrl}/?token=${workflow?.context?.metadata?.token}`,
+        '_blank',
+      );
+    });
+  }, [mutateRevisionCase, pendingWorkflowEvents, workflow]);
```

Comment:
From now on the backoffice sends the revision case level event to an array of workflows (parent and children).